backend/app/assets/javascripts/spree/backend/templates/variants/autocomplete_stock.hbs

```diff
@@ -11,7 +11,7 @@
         <th>{{ t "location" }}</th>
         <th>{{ t "count_on_hand" }}</th>
         <th>{{ t "quantity" }}</th>
-        <th class="actions"></th>
+        <th class="actions">{{ t "actions" }}</th>
```

Comment:
I'm not sure how does handlebars know to translate this correctly?

I haven't seen any translations for, for example: `{{ t "select_stock" }}` on line 2